docs/src/scheduling.md

```diff
@@ -18,22 +18,18 @@ section in the definition of the cluster, which supports:
 - node selectors
 - tolerations
 
-!!! Info
-    CloudNativePG does not support pod templates for finer control
-    on the scheduling of workloads. While they were part of the initial concept,
-    the development team decided to postpone their introduction in a newer
-    version of the API (most likely v2 of CNPG).
+## Pod Affinity and Anti-Affinity
 
-## Pod affinity and anti-affinity
+Kubernetes provides mechanisms to control where pods are scheduled using
+*affinity* and *anti-affinity* rules. These rules allow you to specify whether
+a pod should be scheduled on particular nodes (*affinity*) or avoided on
+specific nodes (*anti-affinity*) based on the workloads already running there.
+This capability is technically referred to as **inter-pod
+affinity/anti-affinity**.
 
-Kubernetes allows you to control which nodes a pod should (*affinity*) or
-should not (*anti-affinity*) be scheduled, based on the actual workloads already
-running in those nodes.
-This is technically known as **inter-pod affinity/anti-affinity**.
-
-CloudNativePG by default will configure the cluster's instances
-preferably on different nodes, while pgBouncer may still run on the same nodes,
-resulting in the following `affinity` definition:
+By default, CloudNativePG configures cluster instances to preferably be
+scheduled on different nodes, while `pgBouncer` instances might still run on
+the same nodes. This results in the following `affinity` configuration:
```

Comment:
It took me a while to understand this.
Here we're saying:

This results:  <affinity snippet>
Given the following cluster specification <cluster yaml>

IMO it would be much clearer if we reversed the flow:

Given the following cluster specification: <cluster yaml>
The resulting affinity rules applied would be: <affinity snippet>

Reply:
Coming out with a change here in the PR